tests/test_sqs.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 
+pytestmark = pytest.mark.anyio
```

Comment:
is this necessary?  Can't this be done globally?

Reply:
I suppose I could try with the `pytest_generate_tests` hook. I'll let you know how that worked out.

Reply:
ya, these look kind of like sprinkled yuk (internally we have another name, lol)

Reply:
ok, figured out how to do it, see comment I recently added